lit_gpt/config.py

```diff
@@ -590,9 +590,26 @@ def norm_class(self) -> Type:
         norm_eps=1e-5,
         _mlp_class="LLaMAMLP",
         intermediate_size=28672,
+    ),
+    
+    # https://huggingface.co/stabilityai/stablecode-completion-alpha-3b/blob/main/config.json
+    dict(
+        org="stabilityai",
+        name="stablecode-completion-alpha-3b",
```

Comment:
You could also add configs for https://huggingface.co/stabilityai/stablecode-instruct-alpha-3b and https://huggingface.co/stabilityai/stablecode-completion-alpha-3b-4k